formats/objfile/reader.go

```
@@ -121,3 +121,12 @@ func (r *Reader) Close() (err error) {
 
 	return
 }
+
+func (r *Reader) FillObject(obj core.Object) error {
+	obj.SetType(r.header.t)
+	obj.SetSize(r.header.size)
+	w, _ := obj.Writer()
```

Please, don't ignore errors.